python/setup.cfg

```diff
@@ -28,26 +28,4 @@ install_requires = cffi>=1.0.0
 opentracing = opentracing; jaeger_client
 
 [flake8]
-# TODO(PROTON-2322) decrease the limit
-max-line-length = 150
-
-# TODO(PROTON-2322) re-enable all these warnings
-ignore =
-    # do not use bare 'except'
-    E722,
-    # imported but unused
-    F401,
-    # ambiguous variable name 'l'
-    E741,
-    # local variable 'int_nodes' is assigned to but never used
-    F841,
-    # TODO(PROTON-2322) decide which of these two warnings we want disabled
-    #  _transport.py:908:21: W503 line break before binary operator
-    #  _transport.py:907:56: W504 line break after binary operator
-    W504,
```

Comment:
One of the w503, w504 should remain disabled, imo, otherwise it's not possible to have line breaks in expressions, which can be useful. But if it works out for you without this, then I guess it's fine.

Reply:
https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
I rewrote the code to avoid using more than one line (and I think made it a little easier to understand)

And in general it's avoidable if you're checking before committing the change.